.drone.star

```diff
@@ -485,22 +487,20 @@ def owncloudService():
     }]
 
 def testMiddleware():
+    environment = {
+        "BACKEND_HOST": "http://owncloud",
+        "NODE_TLS_REJECT_UNAUTHORIZED": "0",
+        "MIDDLEWARE_HOST": "testmiddleware",
+    }
+
     return [{
-        "name": "testmiddleware",
-        "image": "owncloudci/nodejs:14",
-        "pull": "always",
-        "environment": {
-            "MIDDLEWARE_HOST": "testmiddleware",
-            "BACKEND_HOST": "http://owncloud",
-        },
-        "commands": [
-            ". ./.drone.env",
-            "git clone https://github.com/owncloud/owncloud-test-middleware.git /drone/src/middleware",
-            "cd /drone/src/middleware",
-            "git checkout $MIDDLEWARE_COMMITID",
-            "yarn install",
-            "yarn start",
-        ],
+        "name": "middleware",
+        "image": OC_TESTING_MIDDLEWARE,
+        "environment": environment,
+        "volumes": [{
+            "name": "uploads",
+            "path": "/filesForUpload",
```

Comment:
```suggestion
            "path": "/usr/src/app/filesForUpload",
```

Reply:
Still getting the error `can't create '/uploads/'single'quotes.txt': No such file or directory` https://drone.owncloud.com/owncloud/client/10546/5/2